Issues building coq-elpi.1.19.3 on Windows (Coq Platform. cygwin MinGW cross)

I have issue sbuilding coq-elpi.1.19.3 in Coq Platform for Windows.

I added a dune patch which fixed most Windows build issues, but this remains.

The error message is:
```
OCAMLLIBDEP src/elpi_coercion_plugin.mlpack
COQDEP VFILES
FILL .merlin
CAMLOPT -c -for-pack Elpi_coercion_plugin src/coq_elpi_coercion_hook.ml
ocamlfind: Package `coq-elpi.elpi' not found
make[3]: *** [Makefile.coq:757: src/coq_elpi_coercion_hook.cmx] Error 2
make[2]: *** [Makefile.coq:409: all] Error 2
make[1]: *** [Makefile:15: build] Error 2
```

What I don't quite understand is how ocamlfind could possibly find a coq-elpi package before coq-elpi is installed.

I have:
```
$ ocamlfind list | grep elpi
elpi                (version: 1.17.4)
elpi.lexer_config   (version: 1.17.4)
elpi.parser         (version: 1.17.4)
elpi.trace          (version: n/a)
elpi.trace.ppx      (version: 1.17.4)
elpi.trace.runtime  (version: 1.17.4)
elpi.util           (version: 1.17.4)
```

Is there a mechanism that ocamlfind looks for packages in the local build folders?

--- a/apps/coercion/Makefile.coq.local
+++ b/apps/coercion/Makefile.coq.local
@@ -1,3 +1,10 @@
 CAMLPKGS+= -package coq-elpi.elpi
-OCAMLPATH:=../../src/:$(OCAMLPATH)
+
+ifeq "$(shell which cygpath >/dev/null 2>&1)" ""
+OCAMLFINDSEP=:
+else
+OCAMLFINDSEP=;
+endif
+
+OCAMLPATH:=../../src/$(OCAMLFINDSEP)$(OCAMLPATH)
 export OCAMLPATH

--- a/apps/tc/Makefile.coq.local
+++ b/apps/tc/Makefile.coq.local
@@ -1,5 +1,12 @@
 CAMLPKGS+= -package coq-elpi.elpi
-OCAMLPATH:=../../src/:$(OCAMLPATH)
+
+ifeq "$(shell which cygpath >/dev/null 2>&1)" ""
+OCAMLFINDSEP=:
+else
+OCAMLFINDSEP=;
+endif
+
+OCAMLPATH:=../../src/$(OCAMLFINDSEP)$(OCAMLPATH)
 export OCAMLPATH
 
 install-extra::

